guides/release/components-update/component-arguments-and-html-attributes.md

```diff
@@ -0,0 +1,154 @@
+If you want to _reuse_ a component in multiple places, you'll need a way to
+template out parts of it.
+
+Let's start with two similar but not identical avatar components, that represent
+different users:
+
+```handlebars {data-filename="app/components/received-message/avatar.hbs"}
+<aside>
+  <div class="avatar" title="Tomster's avatar">T</div>
+</aside>
+```
+
+```handlebars {data-filename="app/components/sent-message/avatar.hbs"}
+<aside class="current-user">
+  <div class="avatar" title="Zoey's avatar">Z</div>
+</aside>
+```
+
+The _structure_ of these components is identical, but they have somewhat
+different content (the user's initial) and attributes (the `title` and `class`
+attributes).
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-message">
+        You may have noticed that the <code>is-active</code> class on the
```

Comment:
I'm not entirely sure if this was intentional. @pzuraq?

Reply:
It was, since this was an aside clarifying the content on the page, and not adding additional but ultimately unrelated content. I believe we have a few examples of that, but maybe it's just something I made up, feel free to change it, but I don't think it should be a Zoey says necessarily.